.deploy.yaml

```diff
@@ -0,0 +1,11 @@
+name: springboard-iogt
+user: ubuntu
+buildscript: sideloader/deploy.sh
+postinstall: sideloader/postinstall.sh
+pip: []
+dependencies:
+ - python-yaml
+ - cmake
+ - elasticsearch
```

Comment:
you can remove this from here. We'll be using the ES currently in prod or an existing one in QA